packages/release-please/src/release-please.ts

```diff
@@ -146,11 +149,51 @@ async function createGitHubRelease(
   await Runner.releaser(ghr);
 }
 
+async function doRelease(
```

Comment:
this is rather than `createReleasePR`?

Reply:
Originally, `doRelease` called both `createReleasePR` and `creatGitHubRelease`, but was later refactored to keep them separate. I'll rename it back